provider/README.md

```diff
@@ -168,22 +168,22 @@ Data: `{ "status_changes": [] }`, an array of objects with the following structu
 
 | Field | Type | Required/Optional | Comments |
 | ----- | ---- | ----------------- | ----- |
-| `provider_name` | String | Required | The public-facing name of the Provider |
 | `provider_id` | UUID | Required | A UUID for the Provider, unique within MDS |
-| `vin` | String | Required | Vehicle Identification Number assigned by Provider |
+| `provider_name` | String | Required | The public-facing name of the Provider |
 | `device_id` | UUID | Required | A unique device ID in UUID format |
+| `vehicle_id` | String | Required | Vehicle Identification Number assigned by Provider |
 | `vehicle_type` | Enum | Required | see [vehicle types](#vehicle-types) table |
 | `propulsion_type` | Enum | Required | see [propulsion types](#propulsion-types) table |
```

Comment:
Shouldn't we add an array here?

Reply:
A VIN is a 17 character string.  Not sure why it would need an array?   https://www.thezebra.com/insurance-news/2130/vehicle-identification-number-101/

Reply:
@hunterowens did you mean to comment on the `propulsion_type` field, which we discussed might be more than one type?

Reply:
20755c6